internal/locate/region_cache.go

```diff
@@ -806,10 +806,11 @@ func (c *RegionCache) GetTiFlashComputeRPCContextByConsistentHash(bo *retry.Back
 
 // KeyLocation is the region and range that a key is located.
 type KeyLocation struct {
-	Region   RegionVerID
-	StartKey []byte
-	EndKey   []byte
-	Buckets  *metapb.Buckets
+	Region    RegionVerID
+	StartKey  []byte
+	EndKey    []byte
+	Buckets   *metapb.Buckets
+	RawRegion *Region
```

Comment:
Now this struct looks very strange. The other fields can be get from `RawRegion`...

Reply:
That's right, but I don't want to break the compatibility...

Reply:
Then, it adds something strange that cannot be deleted forever in the future for compatibility...

Now, I think we can just use `GetTiKVRPCContext` for the information we want. We can polish the way in the future.

Reply:
`GetTiKVRPCContext` +1, clock is ticking, better to keep it easy to main